libenkf/src/runpath_list.c

```diff
@@ -257,6 +257,7 @@ bool runpath_list_load(runpath_list_type * list) {
       int iens, iter;
 
       int read_count = fscanf(stream, "%d %255s %255s %d", &iens, runpath, basename, &iter);
+      fclose(stream);
```

Comment:
I think this should be after the while loop

Reply:
fixed